### ISSUE TITLE
[alert] Disabling alert doesn't work

### ISSUE THREAD

**Issue by poleszcz** [appoptics/terraform-provider-appoptics#58]
After setting `active` to `false` alert stays enabled in AO and each time plan tries to disable it

```hcl
  # appoptics_alert.example_alert will be updated in-place
  ~ resource "appoptics_alert" "example_alert" {
      ~ active        = true -> false
        id            = "109955884"
        name          = "example-alert"
        # (3 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }
```

**Comment by poleszcz** [comment]
Issue is caused by the way how json encoder handles false booleans...
https://github.com/appoptics/appoptics-api-go/blob/0d52a6e35dd9872c96be9961c0474a8917e59415/alerts.go#L26
https://github.com/golang/go/issues/13284

False booleans are removed from struct and not encoded into http request.

**Comment by poleszcz** [comment]
Fixed with v0.5.2
